### PR TITLE
fix(anoncreds-rs): revocation status list as JSON

### PR DESCRIPTION
**Pull request by genaris**
Now that anoncreds-rs exposes revocation status list from JSON, there should not need in `AnonCredsRsVerifierService` to recreate an object, so here we pass it directly as JSON.

### CODE REVIEW

File: packages/anoncreds-rs/src/services/AnonCredsRsVerifierService.ts [hunk 1]
```diff
@@ -24,27 +24,15 @@ export class AnonCredsRsVerifierService implements AnonCredsVerifierService {
...
-              issuanceByDefault: true,
-              revocationRegistryDefinition: revocationRegistryDefinitions[revocationRegistryDefinitionId],
-              revocationRegistryDefinitionId,
-              timestamp: Number(timestamp),
-            })
-          )
-        }
+        revocationRegistryDefinitions[revocationRegistryDefinitionId] = definition as unknown as JsonObject
+
+        lists.push(...Object.values(revocationStatusLists).map((item) => item as unknown as JsonObject))
```
Comment (blu3beri) [inline packages/anoncreds-rs/src/services/AnonCredsRsVerifierService.ts:35]:
```suggestion
        lists.push((...Object.values(revocationStatusLists)) as unknown as Array<JsonObject>)
```

File: packages/anoncreds-rs/src/services/AnonCredsRsVerifierService.ts [hunk 2]
```diff
@@ -24,27 +24,15 @@ export class AnonCredsRsVerifierService implements AnonCredsVerifierService {
...
-            RevocationStatusList.create({
-              issuerId: definition.issuerId,
-              issuanceByDefault: true,
-              revocationRegistryDefinition: revocationRegistryDefinitions[revocationRegistryDefinitionId],
-              revocationRegistryDefinitionId,
-              timestamp: Number(timestamp),
-            })
-          )
-        }
+        revocationRegistryDefinitions[revocationRegistryDefinitionId] = definition as unknown as JsonObject
```
Comment (blu3beri) [inline packages/anoncreds-rs/src/services/AnonCredsRsVerifierService.ts:33]: Any chance we can improve the casting? If we do `as unknown as...` it is very close to just using any and should be avoided whenever.

**Review by blu3beri — DISMISSED** [review]
Some small comments about the casting.

File: packages/anoncreds-rs/src/services/AnonCredsRsVerifierService.ts (hunk 2 above)
Comment (genaris) [reply packages/anoncreds-rs/src/services/AnonCredsRsVerifierService.ts:33]: Yes, I agree, it's something we are still missing in anoncreds-rs JS API. I think we'll need to complete AnonCreds object modeling in AFJ and replicate it in anoncreds-rs javascript wrapper so they match.

File: packages/anoncreds-rs/src/services/AnonCredsRsVerifierService.ts (hunk 1 above)
Comment (genaris) [reply packages/anoncreds-rs/src/services/AnonCredsRsVerifierService.ts]: Done!

**Review by TimoGlastra — APPROVED** [review]

**Pull request merged**